Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
     - script: |
         cmake --version
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
Don't we need specific VS tool set for Python binaries? Should we keep the CI the same as the packaging? I am setting the following in my SimpleITK builds (but I'm not sure how it would work here):
```
CTEST_CMAKE_GENERATOR_TOOLSET: v141,host=x64
```

Reply:
Ninja picks up CL version from the path, and this sets the x64 architecture I belive.

Reply:
It is a good idea to use an explicit version version of the binaries. We can specify this with arguments to vcvars64.bat. I will add that...